DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/DeactivateAdminConfirmationViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Administrator
+{
+    using DigitalLearningSolutions.Data.Models.User;
+
+    public class DeactivateAdminConfirmationViewModel
+    {
+        public DeactivateAdminConfirmationViewModel(){}
```

Comment:
This view model is not actually used in the view. Seeing as the view is very generic and does not require anything to be passed in via a model, this class can be safely deleted.

Reply:
Viewmodel class is removed as suggested _